### DISCUSSION TITLE
A lower resolution setting

### DISCUSSION THREAD

**Discussion by Joemama45** [skyline-emu/skyline#1422] [Ideas]
If it's possible, you guys in the team could add this option to make games with high graphics that run at lower frames per second, run faster in a lower resolution, that way, it's more compatible with more devices. 

Best of luck to the team of skyline!!!

**Comment by yhak22** [comment]
Maybe in the future they will do that.
But for now we need a lot of games to boot Ingame and yes even 3d games.
So we just wait untill that happen.

**Reply by Joemama45** [reply]
You are right, there are some 3d games that run ok, but it does need more time

**Comment by ghost** [comment]
Omg. Hey. I'm so sorry. Now everything is starting to make sense. My son
must have signed me up for this app and site out of anger towards me,
because I took his game system away . It was inferring with his school
work. Once again .. I'm sorry you guys. And good luck on the game
programing... But how do I delete my unahrurized acct my son was kind
enough to sign me up for knowing I don't know anything about this stuff. I
just want to delete my account , and leave you all who who's what your
doing. Handle ya bussiness.

**Reply by bylaws** [reply]
@Wimgman78 
This link has details on deleting your account:
https://docs.github.com/en/enterprise-server@3.2/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/deleting-your-personal-account#:~:text=In%20the%20upper-right%20corner
